vello_tests/tests/smoke_snapshots.rs

```diff
@@ -112,3 +112,31 @@ fn two_emoji_gpu() {
 fn two_emoji_cpu() {
     two_emoji(true);
 }
+
+#[test]
+#[cfg_attr(skip_gpu_tests, ignore)]
+fn test_bgra() {
+    let mut scene = Scene::new();
+    let mut blob: Vec<u8> = Vec::new();
+    [
+        palette::css::RED,
+        palette::css::LIME,
+        palette::css::BLUE,
+        palette::css::WHITE,
+    ]
+    .iter()
+    .for_each(|c| {
+        blob.extend(c.to_rgba8().to_u8_array());
+    });
+    let image = ImageBrush::new(ImageData {
+        data: blob.into(),
+        format: ImageFormat::Bgra8,
+        width: 2,
+        height: 2,
+        alpha_type: ImageAlphaType::Alpha,
+    });
+    scene.draw_image(&image, Affine::IDENTITY);
+    smoke_snapshot_test_sync(scene, &TestParams::new("bgra", 2, 2))
+        .unwrap()
+        .assert_mean_less_than(0.01);
```

Comment:
The CI failure seems somewhat spurious. I suspect it's because we use the closeness detection hashing thing, rather than the pixel-wise comparison operator used by the sparse strip tests.

In this case, just increasing the allowed mean is a reasonable thing; I've manually checked the images, and they aren't perceptually different.